core/templates/dev/head/pages/library/search_bar_directive.html

```diff
@@ -1,49 +1,52 @@
-<form class="navbar-form navbar-left oppia-search-bar-form" role="search">
-  <div class="form-group">
-    <div class="input-group" style="max-width: 300px; width: 300px;">
-      <div class="input-group-addon oppia-search-bar-icon">
-        <i class="material-icons md-18" ng-if="!isSearchInProgress()">&#xE8B6;</i>
-        <span ng-if="isSearchInProgress()">
-          <i class="material-icons md-18 oppia-animate-spin">&#xE863;</i>
-        </span>
-      </div>
-      <input type="text" class="form-control oppia-search-bar-input oppia-search-bar-text-input protractor-test-search-input"
+<div class="oppia-same-row">
```

Comment:
I don't really understand the name of this class. Was it not in the same row before?